tools/CredScanSuppressions.json

```diff
@@ -125,6 +125,10 @@
         {
             "file": "\\src\\ResourceManager\\Compute\\Stack\\Commands.Compute.Test\\Templates\\tstorgnztn-validator.pem", 
             "_justification": "Legitimate test certificate file"
+        },
+        {
+            "file": "\\src\\ResourceManager\\Sql\\Commands.Sql.Test\\ScenarioTests\\TDECertificateTests.ps1", 
+            "_justification": "Test passwords only valid for the duration of the test"
         }
```

Comment:
@nivimsft 
The preferred way to avoid false positives matches is to use [inline-suppressions](https://microsoft.sharepoint.com/teams/CESecEngineering/CredScan/CredScan%20Wiki/Suppression%20Usage%20Examples.aspx).